public/electron.ts

```diff
@@ -112,10 +129,10 @@ class Main {
     private static registerHandler(channel: string, handler: (...args: any[]) => any) {
         ipcMain.handle(channel, async (...args) => {
             try {
-                return {result: await Promise.resolve(handler(...args))};
+                return { result: await Promise.resolve(handler(...args)) };
             } catch (e) {
-               const error = formatError(e);
-               return {error};
+                const error = formatError(e);
```

Comment:
do we need a space here?

Reply:
Added by VSCode hitting Alt+Shift+F to autoformat the document. It *does* line the `const` vertically with the `return` two lines previously.